docs/user-guide/getting-started.md

```diff
@@ -18,6 +18,8 @@ Without RBAC, you can use:
 kubectl apply -f https://getambassador.io/yaml/ambassador/ambassador-no-rbac.yaml
 ```
 
+Note: To check if Kubernetes has RBAC enabled, run `kubectl cluster-info dump --namespace kube-system | grep authorization-mode`. If you see something like `--authorization-mode=Node,RBAC` in the output, then RBAC is enabled.
+
 For production configurations, we recommend you download these YAML files as your starting point, and customize them accordingly (e.g., your namespace). 
```

Comment:
I'd put this after the "For production configurations" ... weird to have 2 notes separated by a paragraph of non-note text.

Reply:
I agree with @richarddli -- move the check to right _before_ the choice of which file to use, so they can use the information to get the correct file, rather than using it to find out that they picked the wrong one. ;)

Reply:
@richarddli @kflynn ack, I've made some changes, PTAL